Tests/Unit/EventListener/AttachRefreshTokenOnSuccessListenerTest.php

```diff
@@ -298,6 +298,10 @@ public function testAttachesTheTokenToTheResponseBodyOnCredentialsAuth()
 
     public function testDoesNothingWhenThereIsNotAUser()
     {
+        if ((new \ReflectionClass(AuthenticationSuccessEvent::class))->getMethod('getUser')->hasReturnType()) {
```

Comment:
This check is necessary because the WIP `lexik/jwt-authentication-bundle` 3.x branch adds return types on this event class, and its `getUser()` method isn't documented as nullable in the first place.  For now I just skip the test when that branch gets installed as it's the only failure when I force all dev dependencies to be pulled in (merging #338 as early as it was might've been premature given we don't look to be anywhere near a stable release yet).